### PR TITLE
Fix pyunit_glrm_grid_user_y.py failure

### PR DESCRIPTION
**Pull request by wendycwong**
I need to check that the following archetypes are different.  Instead of just looking at one pair, I compare all pairs of archetypes.

### CODE REVIEW

File: h2o-py/tests/testdir_algos/glrm/pyunit_glrm_grid_user_y.py [hunk 1]
```diff
@@ -54,9 +55,32 @@ def glrm_grid_user_y():
...
+                           or not(pyunit_utils.equal_two_arrays(model1Archetypes[1], model2Archetypes[1], throw_error=False)) \
+                           or not(pyunit_utils.equal_two_arrays(model1Archetypes[2], model2Archetypes[2], throw_error=False)) \
+                           or not(pyunit_utils.equal_two_arrays(model1Archetypes[3], model2Archetypes[3], throw_error=False)) \
+                           or not(pyunit_utils.equal_two_arrays(model1Archetypes[4], model2Archetypes[4], throw_error=False)) \
+                           or not(pyunit_utils.equal_two_arrays(model1Archetypes[5], model2Archetypes[5], throw_error=False)) \
+                           or not(pyunit_utils.equal_two_arrays(model1Archetypes[6], model2Archetypes[6], throw_error=False)) \
+                           or not(pyunit_utils.equal_two_arrays(model1Archetypes[7], model2Archetypes[7], throw_error=False)) \
+                           or not(pyunit_utils.equal_two_arrays(model1Archetypes[8], model2Archetypes[8], throw_error=False)) \
+                           or not(pyunit_utils.equal_two_arrays(model1Archetypes[9], model2Archetypes[9], throw_error=False))
```
Comment (sebhrusen) [inline h2o-py/tests/testdir_algos/glrm/pyunit_glrm_grid_user_y.py:69]:
```suggestion
    archetypesNotEqual12 = not all([pyunit_utils.equal_two_arrays(model1Archetypes[i], model2Archetypes[i], throw_error=False) for i in range(10)])
```

File: h2o-py/tests/testdir_algos/glrm/pyunit_glrm_grid_user_y.py [hunk 2]
```diff
@@ -54,9 +55,32 @@ def glrm_grid_user_y():
...
+                           or not(pyunit_utils.equal_two_arrays(model3Archetypes[1], model2Archetypes[1], throw_error=False)) \
+                           or not(pyunit_utils.equal_two_arrays(model3Archetypes[2], model2Archetypes[2], throw_error=False)) \
+                           or not(pyunit_utils.equal_two_arrays(model3Archetypes[3], model2Archetypes[3], throw_error=False)) \
+                           or not(pyunit_utils.equal_two_arrays(model3Archetypes[4], model2Archetypes[4], throw_error=False)) \
+                           or not(pyunit_utils.equal_two_arrays(model3Archetypes[5], model2Archetypes[5], throw_error=False)) \
+                           or not(pyunit_utils.equal_two_arrays(model3Archetypes[6], model2Archetypes[6], throw_error=False)) \
+                           or not(pyunit_utils.equal_two_arrays(model3Archetypes[7], model2Archetypes[7], throw_error=False)) \
+                           or not(pyunit_utils.equal_two_arrays(model3Archetypes[8], model2Archetypes[8], throw_error=False)) \
+                           or not(pyunit_utils.equal_two_arrays(model3Archetypes[9], model2Archetypes[9], throw_error=False))
```
Comment (sebhrusen) [inline h2o-py/tests/testdir_algos/glrm/pyunit_glrm_grid_user_y.py:82]: same as above

**Review by sebhrusen — COMMENTED** [review]
you can probably simplify the condition

File: h2o-py/tests/testdir_algos/glrm/pyunit_glrm_grid_user_y.py (hunk 2 above)
Comment (wendycwong) [reply h2o-py/tests/testdir_algos/glrm/pyunit_glrm_grid_user_y.py]: Clever!  Thanks.

**Review by sebhrusen — APPROVED** [review]
thanks @wendycwong

**Pull request merged**